sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/BeamSqlEnv.java

```diff
@@ -194,14 +198,18 @@ public BeamSqlEnvBuilder setQueryPlannerClassName(String name) {
       return this;
     }
 
+    public void setPipelineOptions(PipelineOptions pipelineOptions) {
```

Comment:
make it part of the builder? I.e. return `this` similar to other methods here, so that it can be used in a fluent call chain